Cargo.toml

```diff
@@ -139,6 +139,7 @@ cms = "0.2.2"
 convert_case = "0.6.0"
 dpe = { path = "dpe/dpe", default-features = false, features = ["dpe_profile_p384_sha384"] }
 crypto = { path = "dpe/crypto", default-features = false }
+crystals-dilithium = "1.0.0"
```

Comment:
This crate is GPL-3.0; is that a problem for us?

https://crates.io/crates/crystals-dilithium

Reply:
Hmm I suppose that's a problem.
Maybe https://crates.io/crates/fips204 can be used, but the sizes of the keys are bit different as dilithium is not map exactly to fips204 even though it's based on it. I guess it needs to match what the HW is going to end up being.

Alternatively https://crates.io/crates/pqcrypto can be used which is a wrapper around a C library.

Reply:
The hardware should be implementing FIPS 204 ML-DSA-87. so i think the FIPS 204 crate is a good option. and the license is compatible.

Reply:
Done